dex/testing/dcrdex/README.md

```diff
@@ -22,6 +22,13 @@ arguments when executing the harness script e.g.:
 ./harness.sh --pgpass=dexpass
```

Comment:
I'm having trouble seeing how this incantation actually does anything. 
`./harness.sh --pgpass=dexpass`
Where are these arguments being parsed into `PG_PASS` or otherwise appended to dcrdex.conf?

Reply:
https://github.com/decred/dcrdex/blob/f2f78d32a833516f8ae0a79d3f31ad25df93e2a0/dex/testing/dcrdex/harness.sh#L162

See also https://github.com/decred/dcrdex/pull/543

Reply:
I think this is a problem then.  The command that gets executed is nonsensical:

```sh
dcrdex --appdata=/home/jon/dextest/dcrdex 2000 --pgpass=dexpass; tmux wait-for -S donedex
```

jessevdk/go-flags silently ignore the "2000" there, but I think we should address this.

Reply:
I certainly carry some responsibility for this confusion. When I added the custom bcast timeout, I had forgotten you could even pass other arguments. 

@itswisdomagain, if you'd like to just remove the custom bcast timeout, feel free. I can revisit it again next time I need it.

Reply:
Replaced cli arg for custom epoch duration with `EPOCH` env variable.